input/templates/partials/catalog/pdp/reserve-in-store-btn.hbs

```diff
@@ -0,0 +1,4 @@
+<button type="submit" class="btn reserve-in-store-btn">
+  <img class="store-thumb" src="{{@root.meta.assetsPath}}img/store-icon-29-29px.png" alt="{{i18n "catalog:reserve.title"}}">
+  {{#if text}}<span class="hidden-xs">{{i18n "catalog:reserve.title"}}</span>{{/if}}
```

Comment:
as we have the same .hbs file for the mix&match page and PDP, we don't need this file anymore